mkosi/config.py

```diff
@@ -363,6 +363,13 @@ def __bool__(self) -> bool:
         return self != BuildSourcesEphemeral.no
 
 
+class Verity(StrEnum):
+    yes = enum.auto()
```

Comment:
Let's name this `signed`

Reply:
It's okay to break existing users of `--verity=yes` ?

Reply:
`parse_boolean()` will take care of the boolean values